applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -851,28 +432,19 @@ static void format_final_result(char *buf, size_t buf_len, size_t buf_max_len)
 	}
 }
 
-static void cmd_send(struct k_work *work)
+static void cmd_send(uint8_t *buf, uint16_t cmd_length, uint16_t buf_size)
 {
 	int err;
 
-	ARG_UNUSED(work);
-
-	uart_receive_disable(false);
+	LOG_HEXDUMP_DBG(buf, cmd_length, "RX");
 
-	if (at_buf_overflow) {
-		rsp_send_error();
-		goto done;
-	}
-
-	LOG_HEXDUMP_DBG(at_buf, at_buf_len, "RX");
-
-	if (cmd_grammar_check(at_buf, at_buf_len) != 0) {
+	if (cmd_grammar_check(buf, cmd_length) != 0) {
 		LOG_ERR("AT command invalid");
 		rsp_send_error();
 		goto done;
```

Comment:
Why use gotos instead of return?

Reply:
Oversight when removing old code, fixed.